src/app/shell/personal-cabinet/provider-admins/provider-admins.component.html

```diff
@@ -17,10 +17,10 @@
         <a [routerLink]="['/create-provider-admin', providerAdminRole.admin]"><button mat-menu-item>Адміністратора гуртка</button></a>
       </mat-menu>
     </ng-template>
-    <ng-template [ngIf]="btnView == providerAdminRoleUkr.deputy">
+    <ng-template [ngIf]="btnView == providerAdminRoleUkr.deputy && subrole !== Role.admin">
       <a [routerLink]="['/create-provider-admin', providerAdminRole.deputy]"><button class="btn" mat-raised-button>Додати</button></a>
     </ng-template>
-    <ng-template [ngIf]="btnView == providerAdminRoleUkr.admin">
+    <ng-template [ngIf]="btnView == providerAdminRoleUkr.admin && subrole !== Role.admin">
```

Comment:
Role.admin is confusing
as I understand that subrole has only provider, correct?
and you are comparing this subrole for providers with role.admin (according to the code)
please change the Role.admin naming. Maybe change it to providerAdmin or maybe we should clarify better naming with the backend team